README.md

```diff
@@ -110,6 +110,77 @@ This can be achieved by modifying the DSN to specify the http protocol.
 http://host1:9000,host2:9000/database?dial_timeout=200ms&max_execution_time=60
 ```
 
+Alternatively, use `OpenDB` and specify the interface type.
+
+```go
+conn := clickhouse.OpenDB(&clickhouse.Options{
+	Addr: []string{"127.0.0.1:8123")},
+	Auth: clickhouse.Auth{
+		Database: "default",
+		Username: "default",
+		Password: "",
+	},
+	Settings: clickhouse.Settings{
+		"max_execution_time": 60,
+	},
+	DialTimeout: 5 * time.Second,
+	Compression: &clickhouse.Compression{
+		Method: clickhouse.CompressionLZ4,
+	},
+	Interface: clickhouse.HttpInterface,
```

Comment:
Please rename `HttpInterface` to `InterfaceHTTP`, in go we use all caps for abbrevations and is better to prefix the constant name with type, not postfix.

Also, can we use Protocol for this? I know that Interface is from clickhouse, but protocol is more common. Anyway, I'm ok with Interface.

Reply:
agreed protocol make sense. Noone is using this yet so its change now.

Reply:
@ernado moved to `Protocol` i.e. 
```
type Protocol int

const (
	Native Protocol = iota
	Http
	Https
)
```